docs/CONFIGURATION.md

```diff
@@ -0,0 +1,218 @@
+- **backend.serviceAccount.annotations**: Adnotacje do dodania do konta serwisowego.
+- **backend.serviceAccount.name**: Nazwa konta serwisowego do użycia. Jeśli nie ustawiono i `create` jest ustawione na true, nazwa jest generowana przy użyciu szablonu fullname.
+- **backend.service.type**: Typ usługi do utworzenia dla backendu. Opcje: `ClusterIP`, `NodePort`, `LoadBalancer`.
+- **backend.service.port**: Port, na którym usługa backendu będzie dostępna.
+- **backend.livenessProbe.httpGet.path**: Ścieżka HTTP używana w liveness probe.
+- **backend.livenessProbe.httpGet.port**: Port używany w liveness probe.
+- **backend.readinessProbe.httpGet.path**: Ścieżka HTTP używana w readiness probe.
+- **backend.readinessProbe.httpGet.port**: Port używany w readiness probe.
+- **backend.autoscaling.enabled**: Czy włączyć autoskalowanie dla backendu.
+- **backend.autoscaling.minReplicas**: Minimalna liczba replik dla backendu.
+- **backend.autoscaling.maxReplicas**: Maksymalna liczba replik dla backendu.
+- **backend.autoscaling.targetCPUUtilizationPercentage**: Docelowe zużycie CPU dla autoskalowania.
+- **backend.autoscaling.targetMemoryUtilizationPercentage**: Docelowe zużycie pamięci dla autoskalowania.
+
+## Konfiguracja Frontend
+
+- **frontend.replicaCount**: Liczba replik dla wdrożenia frontendu.
+- **frontend.image.repository**: Repozytorium obrazu Docker dla frontendu.
+- **frontend.image.pullPolicy**: Polityka pobierania obrazu dla frontendu. Opcje: `Always`, `IfNotPresent`, `Never`.
+- **frontend.image.tag**: Tag obrazu Docker używanego dla frontendu. Domyślnie: wersja aplikacji w chart.
+- **frontend.service.type**: Typ usługi do utworzenia dla frontendu. Opcje: `ClusterIP`, `NodePort`, `LoadBalancer`.
+- **frontend.service.port**: Port, na którym usługa frontendu będzie dostępna.
+- **frontend.autoscaling.enabled**: Czy włączyć autoskalowanie dla frontendu.
+- **frontend.autoscaling.minReplicas**: Minimalna liczba replik dla frontendu.
+- **frontend.autoscaling.maxReplicas**: Maksymalna liczba replik dla frontendu.
+- **frontend.autoscaling.targetCPUUtilizationPercentage**: Docelowe zużycie CPU dla autoskalowania.
+- **frontend.autoscaling.targetMemoryUtilizationPercentage**: Docelowe zużycie pamięci dla autoskalowania.
+
+## Konfiguracja Ingress
+
+- **ingress.enabled**: Czy włączyć ingress dla aplikacji.
+- **ingress.className**: Nazwa klasy kontrolera ingress do użycia.
+- **ingress.annotations**: Adnotacje do dodania do zasobu ingress.
+- **ingress.hosts**: Lista hostów dla zasobu ingress. Każdy host może mieć własny zestaw ścieżek.
+- **ingress.tls**: Lista konfiguracji TLS dla zasobu ingress. Każda konfiguracja określa nazwę sekretu i listę hostów.
+
+## Konfiguracja RBAC
+
+- **rbac.create**: Czy utworzyć zasoby RBAC dla aplikacji.
+- **rbac.rules**: Lista reguł RBAC do zastosowania. Każda reguła określa grupy API, zasoby i działania (verbs).
```

Comment:
RBAC configuration is part of the backend, but I won’t ask you to change it.

Reply:
Well backend uses it, but the role itself is not part of backend deployment in the folder structure hence my and LLM confusion. You won't ask me so I am not sure what is the point of this comment, for now I rake no action on this

Reply:
To tell you what I think. I think it should be in backend but I will not enforce this.

Reply:
made the change